python/lsst/eochar/pair_pixelcovar.py

```diff
@@ -0,0 +1,161 @@
+"""
+  Routine derived from James Chiang's pair_stats.py;
+  Adapted to Augustin Guyonnet's algorithm of 
+  pixel-correlation estimate from the diff. of
+  a flat-fields pair.
+  Author : Sylvain Baumont
+
+  Use : $ python pair_pixelcovar.py 
+     => test on simulated images
+        $ python pair_pixelcovar.py flat1.fits flat2.fits
+     => test on existing image files
+"""
+
+import copy
+import numpy as np
+import numpy.random as random
+import pyfits
+from lsst.eotest.pyfitsTools import pyfitsWriteto
+
+import lsst.afw.geom as afwGeom
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+
+import lsst.eotest.image_utils as imutils
+
+from math import sqrt # from afw.Math ?
+import os,sys
+
+keepmask = False
+#keepmask = True # Uncomment this if you wish to examine the mask images (one per ampli.)
+
+varclip = lambda x : afwMath.makeStatistics(x, afwMath.VARIANCECLIP).getValue()
+
+def exptime(infile):
+    try:

```

Comment:
If this were a normal code review, I might put a comment or change request here.